### ISSUE TITLE
Em dash causing ascii error when trying to import.

### ISSUE THREAD

**Issue by Nobrumski** [evilhero/mylar#2219]
**Describe the bug**
When running manual post processing on Captain Marvel - Carol Danvers - The Ms. Marvel Years, I am getting an ascii error. Copy and paste didn't pick it up, but the dash between Danvers and The is actually an Em dash. Changing it manually to a dash fixes it so I don't know if this is something that can be addressed or not in case other series use it on Comicvine. 

**Screenshots/Logs**
27-Feb-2019 20:33:46 - DEBUG :: mylar.listFiles.146 : Thread-31 : [FILENAME]: Captain Marvel - Carol Danvers  The Ms. Marvel Years (Vol.2018) #01.cbz
27-Feb-2019 20:33:46 - DEBUG :: mylar.parseit.226 : Thread-31 : [CORRECTION] Sub-directory found. Altering path configuration.
27-Feb-2019 20:33:46 - DEBUG :: mylar.parseit.228 : Thread-31 : [SUB-PATH] Checking Folder Name for more information.
27-Feb-2019 20:33:46 - DEBUG :: mylar.parseit.230 : Thread-31 : [SUB-PATH] Original Path : Y:\Import
27-Feb-2019 20:33:46 - DEBUG :: mylar.parseit.231 : Thread-31 : [SUB-PATH] Sub-directory : Y:\Import\Captain Marvel - Carol Danvers - The Ms. Marvel Years (2018)
27-Feb-2019 20:33:46 - ERROR :: mylar.excepthook.315 : Thread-31 : Uncaught exception: Traceback (most recent call last):
  File "C:\Mylar\mylar-D\mylar\logger.py", line 337, in new_run
    old_run(*args, **kwargs)
  File "C:\Python27\lib\threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "C:\Mylar\mylar-D\mylar\PostProcessor.py", line 403, in Process
    filelist = flc.listFiles()
  File "C:\Mylar\mylar-D\mylar\filechecker.py", line 147, in listFiles
    runresults = self.parseit(self.dir, filename, filedir)
  File "C:\Mylar\mylar-D\mylar\filechecker.py", line 233, in parseit
    if path in subpath:
UnicodeDecodeError: 'ascii' codec can't decode byte 0x96 in position 41: ordinal not in range(128)

Traceback (most recent call last):
  File "C:\Mylar\mylar-D\mylar\logger.py", line 337, in new_run
    old_run(*args, **kwargs)
  File "C:\Python27\lib\threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "C:\Mylar\mylar-D\mylar\PostProcessor.py", line 403, in Process
    filelist = flc.listFiles()
  File "C:\Mylar\mylar-D\mylar\filechecker.py", line 147, in listFiles
    runresults = self.parseit(self.dir, filename, filedir)
  File "C:\Mylar\mylar-D\mylar\filechecker.py", line 233, in parseit
    if path in subpath:
UnicodeDecodeError: 'ascii' codec can't decode byte 0x96 in position 41: ordinal not in range(128)

**Environment (please complete the following information):**
 - OS: [ Windows10]
 - Branch / commit: [Dev/c13425]

**Additional information**
https://comicvine.gamespot.com/captain-marvel-carol-danvers-the-ms-marvel-years/4050-109441/

**Comment by Nobrumski** [comment]
Iron Fist - Sabretooth – Round Two (2018)
I almost wonder if it might be Comicvine doing it since I just found another one. Notice the Em Dash between Sabretooth and Round.

**Comment by evilhero** [comment]
Yeah leave it to CV to make things interesting...

I can confirm the error and was able to replicate it - just trying to figure out how best to implement the change so it flows thru the module properly and not throw any additional encoding errors.

**Comment by evilhero** [comment]
Should be all fixed up now in the latest development branch. However, if it's not working for you please let me know.

**Comment by Nobrumski** [comment]
Thanks. I came across this again with Walking Dead TPBs. 

I like having something in there when just looking at the file to tell it is just a duplicate file of the actual comic along with the issue or in this case volume number and since they share the same title I figured volume year was best. Since this causes problems with Mylar finding the file after moving, is there anything you might recommend? I don't know if putting the year of publication in parenthesis at the end will also cause post processing to ignore the file.
